### ISSUE TITLE
Addition of “Total installs” to sort options

### ISSUE THREAD

**Issue by neuromusic** [chanzuckerberg/napari-hub#707]

**Comment by liaprins-czi** [comment]
@neuromusic Do we now want "Total installs" to be the default sort order (pre-search)? Before a user searches, the current default sort is alphabetically by plugin name. But now that one will be at the end of the sort order list and Total installs at the top (until user runs a search).

We can also talk about this on Friday at the call you set up.

**Comment by liaprins-czi** [comment]
Designs are here:
https://www.figma.com/file/d3t84J9Uzmj0GZnYROzdQX/Activity-Dashboard?node-id=1624%3A2495

I will update the selected radio button if needed, per above comment.

**Comment by neuromusic** [comment]
> Before a user searches, the current default sort is alphabetically by plugin name.

I believe the current default is "Recently updated", which I think is ideal, since it keeps the list fresh and highlights new activity in the ecosystem.

**Comment by liaprins-czi** [comment]
> the current default is "Recently updated"

Yeah I got mixed up! So just to verify, do we still want to keep "Recently updated" as the default (pre-search) sorting option (or make total install the new default)?
If so, should we re-order the list of sort options so it's at the top (pre-search), e.g. 
1. [Relevance]
2. Recently updated
3. Total installs
4. Newest
5. Alphabetical (formerly Plugin Name)

**Comment by richaagarwal** [comment]
This is blocked as we want to migrate to a database before we take on additional search/filter work for activity dashboard metrics.
